README.md

```diff
@@ -5,11 +5,11 @@
 [![GoDoc](https://godoc.org/github.com/Azure/aks-engine?status.svg)](https://godoc.org/github.com/Azure/aks-engine)
 [![Go Report Card](https://goreportcard.com/badge/github.com/Azure/aks-engine)](https://goreportcard.com/report/github.com/Azure/aks-engine)
 
-AKS Engine is the easiest way to provision a self-managed Kubernetes cluster on Azure.
+AKS Engine is a legacy ARM template-driven way to provision a self-managed Kubernetes cluster on Azure.
```

Comment:
I would phrase it as "AKS Engine is an ARM template-driven way"

I would not make a statement that ARM templates are "legacy" - aks-engine may become legacy but ARM templates have not (yet?) been put onto the legacy pile.  (They may not be the best way to deal with kubernetes clusters but that does not make them legacy yet)

Reply:
Good point. Will reword - happy to take suggestions

Reply:
"AKS Engine is a legacy tool that leverages ARM templates to..."